.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+node_js:
+  - '4'
+  - '6'
+script:
+  - npm run coverage
+  - npm run codecov
+notifications:
+  email:
+    on_success: change
+    on_failure: change
```

Comment:
You need to get an hipchat API token first, and the hipchat room ID. Then you encrypt those with travis CLI and you add the encrypted token in this config.